### DISCUSSION TITLE
Blender 3.3 Will Not Recognize 6900xt for HIP Rendering

### DISCUSSION THREAD

**Discussion by ShiroyukiX** [rocm-arch/rocm-arch#925] [Support] [answered]
Distro: Arch Linux (KDE)
CPU: Ryzen 5900x
GPU: AMD 6900XT (founder's card)

Regardless if I use `hip-runtime-amd` and add `/opt/rocm/bin` to `PATH` environment variable or use `blender-rocm` over the normal package, blender will not recognize my 6900XT nor CPU when selecting the HIP option for cycles rendering. I'm not certain if blender v3.3 is the issue or what.

If there's any log files necessary, please elaborate on the procedure so I can provide them.

**Comment by tpkessler** [comment, marked as answer]
What version of ROCm are you using? Please remove all ROCm packages and reinstall HIP 5.4.0 (just updated).

**Reply by ShiroyukiX** [reply]
Looks like it worked. I also used `paru` as mentioned in the guide over `yay`. Not sure if that makes a difference.
